Firmware/RTK_Everywhere/Begin.ino

```diff
@@ -377,6 +377,11 @@ void beginVersion()
     getFirmwareVersion(versionString, sizeof(versionString), true);
     systemPrintf("SparkFun RTK %s %s\r\n", platformPrefix, versionString);
 
+#if ENABLE_DEVELOPER && defined(DEVELOPER_MAC_ADDRESS)
+    static const uint8_t developerMacAddress[] = {DEVELOPER_MAC_ADDRESS};
+    esp_base_mac_addr_set(developerMacAddress);
```

Comment:
I like this approach but you may want to follow this with a big warning print that the firmware is changing the device's MAC address. A warning like this was helpful when the key/tokens get overwritten (or not).